modules/logging-es-deploy-cli.adoc

```diff
@@ -0,0 +1,336 @@
+<2> A string value that specifies the label as shown to ensure that cluster monitoring scrapes the `openshift-operators-redhat` namespace.
+
+
+. Apply the namespace object YAML configuration that follows for the Red Hat OpenShift Logging Operator:
++
+[source,terminal]
+----
+$ oc apply -f eo-namespace.yaml
+----
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-logging
+  annotations:
+    openshift.io/node-selector: ""
+  labels:
+    openshift.io/cluster-monitoring: "true"
+----
+
+. Apply the Operator Group object YAML configuration that follows:
++
+[source,terminal]
+----
+$ oc apply -f eo-og.yaml
+----
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: openshift-operators-redhat
+  namespace: openshift-operators-redhat <1>
+spec: {}
+----
+<1> You must specify the `openshift-operators-redhat` namespace.
+
+
+.. Apply the Subscription object YAML configuration that follows:
++
+[source,terminal]
+----
+$ oc apply -f eo-sub.yaml
+----
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: "elasticsearch-operator"
+  namespace: "openshift-operators-redhat" <1>
+spec:
+  channel: "stable" <2>
+  installPlanApproval: "Automatic" <3>
+  source: "redhat-operators" <4>
+  sourceNamespace: "openshift-marketplace"
+  name: "elasticsearch-operator"
+----
+<1> You must specify the `openshift-operators-redhat` namespace.
+<2> Specify `stable`, or `stable-5.<y>` as the channel. See the following note.
+<3> `Automatic` allows the Operator Lifecycle Manager (OLM) to automatically update the Operator when a new version is available. `Manual` requires a user with appropriate credentials to approve the Operator update.
+<4> Specify `redhat-operators`. If your {product-title} cluster is installed on a restricted network, also known as a disconnected cluster,
+specify the name of the CatalogSource object created when you configured the Operator Lifecycle Manager (OLM).
++
+[NOTE]
+====
+Specifying `stable` installs the current version of the latest stable release. Using `stable` with `installPlanApproval: "Automatic"`, will automatically upgrade your operators to the latest stable major and minor release.
+
+Specifying `stable-5.<y>` installs the current minor version of a specific major release. Using `stable-5.<y>` with `installPlanApproval: "Automatic"`, will automatically upgrade your operators to the latest stable minor release within the major release you specify with `y`.
+====
+
+
+The OpenShift Elasticsearch Operator is installed to the `openshift-operators-redhat` namespace and copied to each project in the cluster.
```

Comment:
This says the operator is installed in one namespace and copied?  That is not correct.  What is it we are trying to convey here? Are we trying to say it "watches" all namespaces?

Reply:
Removed.